### PR TITLE
feat/standalone-browser-versions-of-all-libaries

### PR DESCRIPTION
**Pull request by Shyam-Raghuwanshi**
- Issue #415

### Author Checklist
- [X] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [X] Vital features and changes captured in unit and/or integration tests
- [X] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [] Extend the documentation, if necessary
- [X] Merging to `main` from `fork:branchname`

### CODE REVIEW

**Comment by mttrbrts** [comment]
Thanks @Shyam-Raghuwanshi . 

@DianaLease @jamieshorten please review.

File: packages/concerto-cto/webpack.config.js [hunk 1]
```diff
@@ -14,57 +14,70 @@
...
+    })
+  );
+
+  config.plugins = plugins;
+
+  return config;
+};
+
+// Rest of the webpack configuration
 module.exports = {
```
Comment (DianaLease) [inline packages/concerto-cto/webpack.config.js:55]: isn't this just going to override the definition on line 22? Should you instead by calling the `override` function here like you did in concerto-util package?

**Review by DianaLease — COMMENTED** [review]
Left a question inline. Also, why is this only needed in these two packages and not concerto-core? Could you add a description with more info to your PR? The proposed solution in the issue didn't seem to involve changing the webpack configs. Thanks!

**Comment by mttrbrts** [comment]
Closing as stale

**Pull request closed without merging**